NewmanPostman/newmantask.ts

```diff
@@ -34,6 +34,20 @@ function GetToolRunner(collectionToRun: string) {
     newman.argIf(typeof reporterHtmlTemplate != 'undefined' && tl.filePathSupplied('reporterHtmlTemplate'), ['--reporter-html-template', reporterHtmlTemplate]);
     let reporterHtmlExport = tl.getPathInput('reporterHtmlExport');
     newman.argIf(typeof reporterHtmlExport != 'undefined' && tl.filePathSupplied('reporterHtmlExport'), ['--reporter-html-export', reporterHtmlExport]);
+    /**
+    * Items for HTML extra https://www.npmjs.com/package/newman-reporter-htmlextra.
+    */
+    let reporterHtmlExtraTemplate = tl.getPathInput('reporterHtmlExtraTemplate', false, true);
+    newman.argIf(typeof reporterHtmlExtraTemplate != 'undefined' && tl.filePathSupplied('reporterHtmlExtraTemplate'), ['--reporter-htmlextra-template ', reporterHtmlTemplate]);
+    let reporterHtmlExtraExport = tl.getPathInput('reporterHtmlExtraExport');
+    newman.argIf(typeof reporterHtmlExtraExport != 'undefined' && tl.filePathSupplied('reporterHtmlExtraExport'), ['--reporter-htmlextra-export', reporterHtmlExtraExport]);
+    let htmlExtraDarkTheme = tl.getBoolInput('htmlExtraDarkTheme');
+    newman.argIf(htmlExtraDarkTheme, ['--reporter-htmlextra-darkTheme']);
+    let htmlExtraTestPaging = tl.getBoolInput('htmlExtraTestPaging');
+    newman.argIf(htmlExtraTestPaging, ['--reporter-htmlextra-testPaging']);
+    let htmlExtraReportTitle = tl.getInput('htmlExtraReportTitle');
+    newman.argIf(htmlExtraReportTitle, ['--reporter-htmlextra-title', htmlExtraReportTitle]);
```

Comment:
@sebcaps does this line look correct to take the title input?

Reply:
@Scott-Emberson I don't exactly remember why but I would rather do it this way for line 49:
`newman.argIf(typeof htmlExtraReportTitle != 'undefined' && htmlExtraReportTitle , ['--reporter-htmlextra-title', htmlExtraReportTitle]);`

It's done this way for other 'Input' arguments. (see `delayRequest` for ex)

Another point which just came in my mind about 'Visible Rule'. This is my suggestion but I'm afraid we'll fall into a problem since reporters is a multiSelect list (cf [this question](https://github.com/microsoft/azure-pipelines-tasks/issues/7711)). Unfortunately it has no clear answer. But I guess  we'd better change strict equal to contains or if impossible remove this rule for the moment (my bad :-( )

Did you try to run existing test to ensure non reg ? (running `npm test`)
@carlowahlstedt did not manage to include test exec in build process, so for the moment we should do it manually. In  second time (ie another PR) it would be great if you could add tests for added options ;-)
Thx

Reply:
Have removed the visibleRule as there is no way from what I can find to do a contains. Have made the change for the htmlExtraReportTitle. 
I have fixed the npm test as it was missing the is-url package. 
In terms of writing a test I am not exactly sure what to write but will have another go in another PR.